Sources/SwiftUIBackports/iOS/PresentationDetents/Detents.swift

```diff
@@ -182,7 +182,15 @@ private extension Backport.Representable {
 
             if let controller = parent?.sheetPresentationController {
                 controller.animateChanges {
-                    controller.detents = detents.map {
+                    controller.detents = detents.sorted(by: {
```

Comment:
Personally I'd prefer to make `Backport.PresentationDetent: Comparable` then implement the static method. That way here would just be `sorted()`

Sound good?